app/controllers/passkit/api/v1/registrations_controller.rb

```diff
@@ -73,7 +73,9 @@ def load_device
         end
 
         def register_device
-          device = Passkit::Device.find_or_create_by!(identifier: params[:device_id]) { |d| d.push_token = push_token }
```

Comment:
In the previous version, no changes are made to the db a device is found. Now, the device is being updated every time. 

Maybe like this to back fill missing push_service_url?
````
          device = Passkit::Device.find_or_create_by!(identifier: params[:device_id]) do |d| 
          d.push_service_url = push_service_url 
          d.push_token = push_token 
          end
          
          device.update(push_service_url: push_service_url) if device.push_service_url.blank?
````